meetings/2021-07-28.md

```diff
@@ -49,30 +49,47 @@ Sarven: Because you're authorized (or not).
 
 Henry: I think we need to go at it from an unauthenticated perspective.
 The first thing we have to cover is how to build apps that work across pods.
-Everyone is using social networks because they're easy to use.
-If we can get everybody to be working on decentralised apps, we're solving a real problem. Not how the CIA deals with it.
+Everyone is using social networks because they are easy to use, have access control and help connect people 
+But, for the moment, these are all done on centralized servers.
```

Comment:
```suggestion
Everyone is using social networks because they are easy to use, have access control, and help connect people; 
but, for the moment, these are all done on centralized servers.
```